sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -295,6 +295,41 @@ case class AlterViewAsCommand(
   }
 }
 
+/**
+ * A command for users to get views in the given database.
+ * If a databaseName is not given, the current database will be used.
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW VIEWS [(IN|FROM) database_name] [[LIKE] 'identifier_with_wildcards'];
+ * }}}
+ */
+case class ShowViewsCommand(
+    databaseName: Option[String],
+    tableIdentifierPattern: Option[String]) extends RunnableCommand {
+
+  // The result of SHOW VIEWS has three basic columns: database, viewName and isTemporary.
+  override val output: Seq[Attribute] = Seq(
+    AttributeReference("namespace", StringType, nullable = false)(),
```

Comment:
This is inconsistent with line 310. At line 310, it's `database`. We should fix here or line 310, @Eric5553 .

BTW, did we decide to use `namespace` here instead of `database`, @cloud-fan and @maropu ?

Reply:
Technically, `ShowTablesCommand` still uses `database` while `ShowTables` uses `namespace`. And, this is `ShowViewsCommand` as a V1 command.

Reply:
> BTW, did we decide to use namespace here instead of database always, @cloud-fan and @maropu ?

If we have a plan to support view APIs in v2 catalog, `namespace` looks ok to me. But, we need to update the other places for consistency, e.g., `ShowTables` https://raw.githubusercontent.com/apache/spark/master/docs/sql-ref-syntax-aux-show-tables.md

Reply:
IMO, as cloud-fan suggested, maybe we should use `namespace` for consistency?  Anyway, I'm changing the comments(line 310) to get along with current code/doc, and waiting for final decision. Thanks.

Reply:
Let's use namespace to avoid potential breaking changes.

`SHOW TABLES` will have breaking change when we fully migrate v1 commands to v2, but that's inevitable. At least we can avoid it for `SHOW VIEWS`.